Khan/perseus
# Remove depricated/unrecommneded widgets from perseus editor dropdown.

## Summary:
Removing deprecated/unrecommended widgets from the perseus editor dropdown by enabling the hidden attribute in the Widget Export value.

Issue: LEMS-2307

## Test plan:

- Go to: https://650db21c3f5d1b2f13c02952-mhmfvxmnkx.chromatic.com/
- Navigate to any perseus editor page (such as ?path=/story/perseuseditor-editor--demo)
- Notice that the following widgets are no longer options.
    - Categorizer
    - Grapher
    - Matrix
    - Measurer
    - Orderer
    - Plotter
    - Group
    - Passage
    - PassageRef
- And "Interactive graph" is now labeled "Interactive graph (Assessments only)"

Before:
![image](https://github.com/user-attachments/assets/3f03b473-be5f-4c75-97d1-54b72e66e716)

After:
![image](https://github.com/user-attachments/assets/796a6a93-3892-4c56-a26f-0e0a03b80b4a)
